src/ConstructiveSolidGeometry/plotting/SurfacePrimitives/ConeMantle.jl

```diff
@@ -1,41 +1,64 @@
-function get_plot_points(c::ConeMantle{T}; n = 30) where {T <: AbstractFloat}
+@recipe function f(cm::ConeMantle, n = 40; subn = 10)
+    e_top = get_top_ellipse(cm)
+    e_bot = get_bot_ellipse(cm)
+    e_top_edges = edges(e_top, n = n)
+    e_bot_edges = edges(e_bot, n = n)
+    linecolor --> :black
+    @series begin
+        label --> "Cone Mantle"
+        e_top_edges
+    end
+    @series begin
+        label := nothing
+        e_bot_edges
+    end
+    @series begin
+        label := nothing
+        map(i -> Edge(e_top_edges[i].a, e_bot_edges[i].a), 1:subn:length(e_top_edges))
+    end
+end
 
-    plot_points = Vector{CartesianPoint{T}}[]
+    # RotZ(π) * -cm.rotation such that the normal vector points inside the cone (Convention)
```

Comment:
I see: please check my comment above, where it was not `RotZ(π) * -t.rotation`.
(Although I do not see how `RotZ(π)` should flip the normal vector...)

Reply:
See reply to comments above ;)